src/Controller/SsoController.php

```diff
@@ -9,6 +9,11 @@
 
 class SsoController extends BaseController
 {
+    public function error($response, $message){
```

Comment:
It would be better if the `error`-function also took in param: `statuscode`. That way the function can be used also for other HTTP Status codes as well.